packages/@aws-cdk/aws-servicecatalog/lib/constraints.ts

```diff
@@ -62,4 +63,41 @@ export interface TagUpdateConstraintOptions extends CommonConstraintOptions {
    * @default true
    */
   readonly allow?: boolean;
+}
+
+/**
+ * An assertion within a template rule, defined by intrinsic functions.
+ */
+export interface Assertion {
+  /**
+   * The assertion condition.
+   */
+  readonly assert: cdk.ICfnConditionExpression;
+
+  /**
+   * The description for the asssertion
+   * @default - no description provided for the assertion.
+   */
+  readonly assertDescription?: string;
```

Comment:
How about calling this just `description`?

Reply:
fixed, should be fine to not get confused with the other description fields.